static/app/views/issueDetails/traceTimeline/traceTimeline.tsx

```diff
@@ -8,7 +8,6 @@ import {space} from 'sentry/styles/space';
 import type {Event} from 'sentry/types/event';
 import useRouteAnalyticsParams from 'sentry/utils/routeAnalytics/useRouteAnalyticsParams';
 import {useDimensions} from 'sentry/utils/useDimensions';
-import useOrganization from 'sentry/utils/useOrganization';
```

Comment:
We don't need to check for the `related-issues-issue-details-page` feature, thus, not needed.